### PR TITLE
Add docs guide for Ubuntu setup

### LINKED ISSUE #483: Guide for setting up a node
## Problem
There may be situations where it is unclear how to quickly set up a node for some testing, development or (later) user-focused purposes. 

## Solution
It may be useful to have a guide for setting up a Trin client from start to finish as a systemd service on Ubuntu.

### PR DESCRIPTION
**Pull request by perama-v**
### What was wrong?

Missing walkthrough for setting up Trin on Ubuntu as noted in #483. This increases friction for newcomers.

Closes #483

### How was it fixed?

Step by step guide in `docs/ubuntu_guide.md` for devs/testers/users to follow.

### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [x] Add entry to the [release notes](https://github.com/ethereum/trin/blob/master/newsfragments/README.md) (may forgo for trivial changes)
- [x] Clean up commit history

### CODE REVIEW

File: docs/ubuntu_guide.md
Comment (ogenev) [inline docs/ubuntu_guide.md:62]: I think the default port 8545 is used for JSON-RPC not for discovery, which `--discovery-port <port>` points to? However, we still can get some conflicts on the TCP 8545 because the default portal json-rpc port is same as eth json-rpc, so changing it with  `--web3-http-address` is reasonable.

File: docs/ubuntu_guide.md
Comment (ogenev) [inline docs/ubuntu_guide.md:65]:
```suggestion
$ sudo ss -tulpn | grep ':8545'
```

File: docs/ubuntu_guide.md
Comment (perama-v) [reply docs/ubuntu_guide.md:62]: Oh you are right, I have conflated discovery and rpc here

**Review by ogenev — APPROVED** [review]
Very useful guide, thanks!

**Comment by perama-v** [comment]
My pleasure. Let me know about next steps if any (I don't have merge access)

**Comment by ogenev** [comment]
> My pleasure. Let me know about next steps if any (I don't have merge access)

Could you squash all or only the first two commits before merging?

**Pull request merged**
